python-meh.spec

```diff
@@ -1,6 +1,3 @@
-%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
-%global with_python3 1
```

Comment:
First it would be nice to have:
``
%global srcname meh
``
and use it later.

Reply:
Sure, although I think just srcname will be enough.